src/main/java/DukeException.java

```diff
@@ -0,0 +1,14 @@
+package main.java;
+
+public class DukeException extends Exception {
```

Comment:
Perhaps DukeException does not need to be public? As DukeException would not be used outside of the package, I think the other exceptions do not need to be public as well